styles/octopus.css

```
@@ -0,0 +1,72 @@
+.octopus {
+    height: 100vw;
+
+    padding-top: 90px;
+
+    background: 
+    linear-gradient(252.43deg, #D45B78 0.01%, #4B3862 101.01%);
+}
+
+.octopus__container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.octopus__img {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: 1280px;
+
+    margin-bottom: 14px;
+}
+
+.octopus__title {
+    color: #FCFCFC;
+
+    margin-bottom: 37px;
+}
+
+.octopus__subtitle {
+    width: 890px;
+    height: 72px;
+
+    text-align: center;
+    color: #F2F2F2;
+
+    margin-bottom: 68px;
+}
+
+.octopus__button {
+    color: #FFFFFF;
+    font-size: 36px;
+    line-height: 42px;
+
+    padding: 20px 59px;
+
+    background-color: #D75C78;
+    border-radius: 5px;
+    box-shadow: 0 5px 15px rgba(0,0,0,0.5);
+}
+
+.octopus__button:hover {
+    background-color: #DF4C6E;
+}
+
+.octopus__button:active {
+    background-color: #C83E5D;
+}
+
+.octopus__bckg-group-bubls {
+    position: absolute;
+    right: 27px;
+    top: 821px;
+}
+
+.octopus__bckg-bubble {
+    position: absolute;
+    left: 128px;
+    top: 775px;
+}
```

```suggestion
.octopus__bubble {
  position: absolute;
}
.octopus__bubble-left {
  left: 128px;
  bottom: ?px;
}
.octopus__bubble {
  position: absolute;
  bottom: ?px;
}
```
Use MODIFICATOR from BEM

---

It doesn't work that way, the pictures get big